common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -117,6 +117,7 @@ def B(*args, **kwargs):
 AUTH_ENTRY_DASHBOARD = 'dashboard'
 AUTH_ENTRY_LOGIN = 'login'
 AUTH_ENTRY_REGISTER = 'register'
+AUTH_ENTRY_ACCOUNT_SETTINGS = 'account_settings'
```

Comment:
@symbolist There are some places in the the code where AUTH_ENTRY_DASHBOARD is referenced, and I wonder if those should change to AUTH_ENTRY_ACCOUNT_SETTINGS.

Specifically, in login_analytics, no 'edx.bi.user.account.linked' event is emitted anymore because the place where the linking happens is no longer the dashboard-- it is account_settings.

Also, reading the comments on change_enrollment, it seems that the usage of AUTH_ENTRY_DASHBOARD in that method should also change to AUTH_ENTRY_ACCOUNT_SETTINGS.

It would be good to have the auto auth changes reviewed by someone familiar with that code-- perhaps @nasthagiri? She is the one who wrote the 2 lines of code I have referenced.

@nasthagiri, for context, the auto-auth linking will no longer occur on the dashboard page, but instead on the account settings page.